piperider_cli/initializer.py

```diff
@@ -84,10 +84,6 @@ def _inherit_datasource_from_dbt_project(dbt_project_path, dbt_profiles_dir=None
     dbt_config.dump(FileSystem.PIPERIDER_CONFIG_PATH)
 
     console.print(f'[[bold green] DBT [/bold green]] Use the existing dbt project file: {dbt_project_path}')
-    console.print(
-        "[[bold green] DBT [/bold green]] "
-        "By default, PipeRider will profile the models and metrics with 'piperider' tag\n"
-        "        Apply 'piperider' tag to your models or change the tag in '.piperider/config.yml'\n")
```

Comment:
In cli.py, we should change the wording:

https://github.com/InfuseAI/piperider/blob/9ed77d1c5ddd25248cfba51b220ae8884a1458c8/piperider_cli/cli.py#L279-L281

no more `tag:piperider`